### PR TITLE
Created a new EarthquakeApplication class that will be used for insta…

### PR DESCRIPTION
**Pull request by alistairholmes**
Created a new Application class for the Instabug reporting. Also added the required dependency :)

### CODE REVIEW

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Unfortunately, you have not registered the `EarthquakeApplication` in the Manifest. Nothing happenes when I take a screenshot or shake the phone.

**Comment by alistairholmes** [comment]
Oh snap! My bad, totally forgot to commit the manifest file.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
There is this error in earthquake_activity.xml:

_Android resource compilation failed
Output:  Quake-Info-master\app\src\main\res\layout\earthquake_activity.xml:26: error: mismatched tag._

There are some issues with the ordering/opening-closing of tags. Check that.

File: app/src/main/java/com/theQuake/quakeInfo/EarthquakeActivity.java [hunk 1]
```diff
@@ -111,6 +113,41 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        swipeLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                // Get a reference to the LoaderManager, in order to interact with loaders.
+                LoaderManager loaderManager = getLoaderManager();
+
+                // Initialize the loader. Pass in the int ID constant defined above and pass in null for
+                // the bundle. Pass in this activity for the LoaderCallbacks parameter (which is valid
+                // because this activity implements the LoaderCallbacks interface).
+                loaderManager.initLoader(EARTHQUAKE_LOADER_ID, null, EarthquakeActivity.this);
```
Comment (Kvaibhav01) [inline app/src/main/java/com/theQuake/quakeInfo/EarthquakeActivity.java:130]: It says wrong third argument type. It requires `LoaderCallbacks` in `EarthquakeActivity.this`.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Your swipe layout had some error in the EarthquakeActivity.java file at line 86:

`swipe = findViewById(R.id.swiperefresh);` There was no id with `swiperefresh`. 

I tested with the screenshot, InstaBug works perfectly! Kudos 🎉. But still, the shake feature doesn't work 😞. Try it with your phone and let me know. Also, check the comment I made on line 130 of EarthquakeActivity file.

**Comment by Kvaibhav01** [comment]
@alistairholmes are you working on this or should I close this PR?

**Comment by Kvaibhav01** [comment]
Closing this PR as no active contribution/activity was noted.

**Pull request closed without merging**